arangosh/Restore/RestoreFeature.cpp

```diff
@@ -968,13 +951,42 @@ arangodb::Result processInputDirectory(
         // reason is that loading into the users collection may change the
         // credentials for the current arangorestore connection!
         usersData = std::move(jobData);
+      } else if (name.isString() && name.stringRef() == "_analyzers") {
+        // special treatment for _analyzers collection - this must be the very first
+        analyzersData = std::move(jobData);
```

Comment:
I guess we should also increase `stats.totalCollections` here, as in the `else` case.